### ISSUE TITLE
Wand auto shoot won't trigger automatically

### ISSUE THREAD

**Issue by IronkneeSS** [dalaranwow/dalaran-wow#5339]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: While having a Staff and Wand equipped, the auto attack won't detect the wand and therefore won't trigger the switch from melee to ranged. (Where, When, What, Why, Who)

**Expected behaviour**: It should auto attack with the wand weapon

**Steps to reproduce the problem**:

1. Right click on a enemy to trigger auto attack, character still attempts to melee
2. 
3. 

**Include proofs for this behaviour**
A video would be needed to prove.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (cre

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
It works as intended. There's an spell called 'Shoot' in your spellbook. [Link 1](http://www.wowhead.com/spell=5019/shoot#comments) - [Link 2](https://us.battle.net/forums/en/wow/topic/3581328395)

Closing up
